google/ggrc-core
# GGRC-6344 Fix mapping of core objects via import

# Issue description

Mapping between some directives are not applied during CSV import.
Affected directives are:
* Standard
* Contract
* Requirement
* Regulation
* Policy

# Steps to test the changes

1. Download import template for a Standard, Policy, Regulation, Contract and Requirement
2. Ensure that the template contains:
   * Standard Import template has "map:regulation", "map:policy", "map:contract", "map:requirement" columns and doesn't have "map:standard" column
   * Policy Import template has "map:regulation", "map:standard", "map:contract", "map:requirement" columns and doesn't have "map:policy" column
   * Regulation Import template has "map:policy", "map:standard", "map:contract", "map:requirement" columns and doesn't have "map:regulation" column
   * Contract Import template has "map:regulation", "map:standard", "map:policy", "map:requirement" columns and doesn't have "map:contract" column
   * Requirement Import template has "map:regulation", "map:standard", "map:policy", "map:contract" and "map:requirement" columns
3. Ensure that mapping between directives correspondingly can be applied by import.

# Solution description

To resolve the issue mapping rules for Relationship has been updated

# Sanity checklist


# PR Review checklist
